apis/types/container_json.go

```diff
@@ -0,0 +1,36 @@
+package types
+
+// ContainerJSON contains response of Remote API:
+// GET "/containers/{name:.*}/json"
+type ContainerJSON struct {
```

Comment:
这个文件可以使用swagger生成么？

Reply:
No. ContainerJSON depends on ContainerState, but I didn't find the definition of ContainerState in swagger.yml.
I will try to fix it.

Reply:
Because some builtin types was not supported by go-swagger, so I can't fix swagger issue now. 
I do have plan to refactor the struct ContainerState & ContainerStatus, and I will fix swagger issue in follow-on PRs.